react/features/base/dialog/components/web/StatelessDialog.tsx

```diff
@@ -280,16 +269,14 @@ class StatelessDialog extends Component<Props> {
         }
     }
 
-    _onSubmit: (?string) => void;
-
     /**
      * Dispatches the action when submitting the dialog.
      *
      * @private
      * @param {string} value - The submitted value if any.
      * @returns {void}
      */
-    _onSubmit(value) {
+    _onSubmit(value?: any) {
```

Comment:
Isn't this a string or undefined?

Reply:
No. It should be either FormEvent or MouseEvent. But because we merged web and native buttons it also needs to be GestureResponderEvent and I thought it would be weird to add native things to web components so I just left it as any for now

Reply:
👍